usaspending_api/conftest.py

```diff
@@ -123,12 +124,21 @@ def django_db_setup(
             ensure_business_categories_functions_exist()
             call_command("load_broker_static_data")
 
-            # temporarily updating the environment so commands in tests will point to the test database
-            old_environ = os.environ.copy()
-            os.environ["DATABASE_URL"] = generate_test_db_connection_string()
+            # The Django function "call_command", which runs a separate process of a django management command, will
+            # use the same settings as calling it normally via commandline. This is fine except when calling it in a
+            # Django test that usually involves the creation/teardown of a test database, and the new management command
+            # will work off the normal database (not the newly created test database). This resolves the issue by
+            # temporarily mocking the DATABASE_URL environment value (in python storage, not actually modifying your
+            # local environment) which will then allow the new python process to accurately point to the test database.
```

Comment:
Are you sure this is the case? I recently looked into this and the `call_command`, called from code, _simulates_ a CLI-call, but actually doesn't start a new process (or at least it didn't look like it did). If it where an actual call to `manage.py cmd_name`, yes that does start a new process. 

If that's the case, it may actually recycle the already parsed settings module, which in this case, would include any substitutions for settings that pytest-django has already done.